src/marketplace/MakeAComponentSaleView.js

```diff
@@ -44,50 +46,57 @@ class MakeAComponentSaleView extends React.Component<Props, State> {
 		//set the users id
 		this.setState({userId:jsonObjectOfUser._id});
 		for (let key in jsonObjectOfUser.inventory.tankComponents) {
-            //This is how many of the current item the user has
-            let amountOfItemsUserHas = jsonObjectOfUser.inventory.tankComponents[key];
-            if(amountOfItemsUserHas > 0)
-            {
-                let obj = {};
-                obj['value'] = key;
-                obj['label'] = key;
-                itemsToSell.push(obj);
-            }		
+		//This is how many of the current item the user has
+			let amountOfItemsUserHas = jsonObjectOfUser.inventory.tankComponents[key];
+			if(amountOfItemsUserHas > 0)
+			{
+				let obj = {};
+				obj['value'] = key;
+				obj['label'] = key;
+				itemsToSell.push(obj);
```

Comment:
:( 👎  this is bad in so many ways...

- obj should be const
- obj is a terrible variable name for some object. What object?
- value and label are very confusing variable names that don't mean anything

Reply:
Forgot a couple:
- no types leads dumb people like me to get very confused when trying to understand what is going on here
- even if you were going to do this, it would be way cleaner to do `obj.value=key;` and `obj.label=key` (but you shouldn't do it like this anyway for the other four reasons)

Reply:
Yeah this is just a toasty review these changes are coming when I handle the infinite request which I grouped together with issue 294

Reply:
okay, sounds good.